dask/dataframe/core.py

```diff
@@ -3227,9 +3227,34 @@ def pivot_table(self, index=None, columns=None,
         return pivot_table(self, index=index, columns=columns, values=values,
                            aggfunc=aggfunc)
 
-    def to_records(self, index=False):
+    def to_records(self, lengths=None, index=False):
```

Comment:
I think lengths should be the last argument. That way we match the pandas signature more closely (though we're missing other kwargs currently).